src/ja/common/proxy/ssh.py

```diff
@@ -1,6 +1,99 @@
+from typing import Dict
+
+from ja.common.message.base import Command, Response
+from ja.common.config import Config
+
+
 class SSHConnection:
     """
-    Establishes an SSH connection to a remote component. Writes Command objects
-    the stdin of Remote objects. Then reads a Response object from stdout of
-    the Remote object. Message objects are read/written as YAML strings.
+    Establishes an SSH connection to a Remote object. Writes Command objects to
+    the stdin of said Remote objects. Then reads a Response object from stdout
+    of the Remote objects. Message objects are read/written as YAML strings.
+    Uses paramiko as the backend for establishing an ssh connection.
```

Comment:
Is the usage of paramiko really relevant to anybody reading this comment? We can mention it in the design document in a page "external libraries" or something like that.

Reply:
It is relevant to our design because this makes it clear that we do not have to describe lower-level classes.